test/sql/bgw_db_scheduler.sql

```diff
@@ -103,11 +108,30 @@ SELECT * FROM _timescaledb_internal.bgw_job_stat;
 -- empty
 SELECT * FROM sorted_bgw_log;
 
+--
+-- Test running the scheduler with a job marked as unscheduled
+--
+
+TRUNCATE bgw_log;
+SELECT ts_bgw_params_reset_time();
+SELECT insert_job('unscheduled', 'bgw_test_job_1', INTERVAL '100ms', INTERVAL '100s', 5, INTERVAL '1s',scheduled:= false);
+SELECT ts_bgw_db_scheduler_test_run_and_wait_for_scheduler_finish(50);
+-- empty
+SELECT * FROM _timescaledb_internal.bgw_job_stat;
+
+SELECT test_toggle_scheduled(1000);
```

Comment:
It toggles from `false` to `true`, while a test from `true` to `false` seems to be missing.

Reply:
the current testing is flawed in that it doesnt actually test that cache invalidations happen but fixing that and adding more tests is outside the scope of this PR

Reply:
Can you link the issue, which will resolve this issue?